.github/workflows/upd-xna-client.yml

```diff
@@ -0,0 +1,161 @@
+# To be able use this workflow you must ensure that your repository have enable write and right access for GitHub Actions.
+#
+# GUI Reference to enable access:
+# Repository --> Actions --> General --> Workflow permissions --> Read and write permissions
+#                                                            \--> Allow Github Actions to create and approve pull requests
+
+name: Update XNA CnCNet Client
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *"
+
+env:
+  REPO_NAME: cncnet-client-mod-base
+  CLIENT_PATH: ClientFiles
+  UPDATE_BRANCH_NAME: update-client-binaries
+  COMMIT_MESSAGE: Update client binaries to the latest version
+  PR_TITLE: Update client binaries to the version
+  PR_BODY: This is an automatic pull request to update client binaries triggered from CnCNet/xna-cncnet-client repository.
+
+jobs:
+  publish-update-pr:
+    runs-on: windows-latest
+
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v3
+      with:
+        lfs: true
+    
+    - name: Download Latest Release
+      id: download
+      uses: robinraju/release-downloader@v1
+      with:
+        repository: CnCNet/xna-cncnet-client
+        latest: true
+        fileName: xna-cncnet-client*
+
+    - name: Open PR
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        $currentVersion = (Get-Item -Path "D:\a\${{ env.REPO_NAME }}\${{ env.REPO_NAME }}\Resources\clientdx.exe").VersionInfo.FileVersion -Replace "\.0$", ""
+        $downloadVersion = '${{ steps.download.outputs.tag_name }}'
+        
+        echo "Current client version: '$currentVersion'"
+        echo "Downloaded client version: '$downloadVersion'"
+
+        if ($currentVersion -eq $downloadVersion)
+        {
+          echo "Update doesn't required"
+        } else
+        {
+          echo "Update required"
+          
+          # Configure git
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+          # Remove old branch if exist and make new one
+          git branch -D ${{ env.UPDATE_BRANCH_NAME }}
+          git checkout -b ${{ env.UPDATE_BRANCH_NAME }}
+        
+          # Delete old binaries, unpack archive, remove archive
+          rm -r -fo "D:\a\${{ env.REPO_NAME }}\${{ env.REPO_NAME }}\Resources\Binaries"
```

Comment:
duplicated repo name in path here and in other places, is that correct?

Reply:
> duplicated repo name in path here and in other places, is that correct?

provided workflow by me works fine, tested in my other client repos. this is a absolute path to the directory